structuredefinitions/UKCore-Observation.xml

```diff
@@ -58,24 +58,68 @@
-	</element>
-	<element id="Observation.extension:triggeredByR5">
-		<path value="Observation.extension"/>
-		<sliceName value="triggeredByR5"/>
-		<type>
-			<code value="Extension"/>
-			<profile value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Observation.triggeredBy"/>
-		</type>
-	</element>
-	<element id="Observation.identifier.assigner">
+      <path value="Observation.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Observation.extension:triggeredByR5">
+      <path value="Observation.extension" />
+      <sliceName value="triggeredByR5" />
+      <short value="Triggering observation(s). This is a R5 backport." />
+      <min value="0" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Observation.triggeredBy" />
+      </type>
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension">
+      <path value="Observation.extension.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="2" />
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension:observation">
+      <path value="Observation.extension.extension" />
+      <sliceName value="observation" />
+      <min value="1" />
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension:observation.value[x]">
+      <path value="Observation.extension.extension.value[x]" />
+      <short value="Triggering observation." />
+      <definition value="A reference to the triggering observation." />
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension:type">
+      <path value="Observation.extension.extension" />
+      <sliceName value="type" />
+      <min value="1" />
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension:type.value[x]">
+      <path value="Observation.extension.extension.value[x]" />
+      <short value="The type of trigger. Reflex | Repeat | Re-run" />
+      <definition value="The type of trigger. Reflex | Repeat | Re-run" />
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension:reason">
+      <path value="Observation.extension.extension" />
+      <sliceName value="reason" />
+      <min value="0" />
+    </element>
+    <element id="Observation.extension:triggeredByR5.extension:reason.value[x]">
+      <path value="Observation.extension.extension.value[x]" />
+      <short value="Reason that the observation was triggered." />
+      <definition value="Provides the reason why this observation was performed as a result of the observation referenced." />
+    </element>
```

Comment:
Did you mean to add all of this. This is different to how you did it last time?

Reply:
Yeah - this ties up with the why i did it in FamilyMemberHistory - for complex extensions, Forge adds the slicing/naming, and then you apply the description on the value[x] of the slice